adding more background info (#61)
fhdsl/AI_for_software: 01-intro.Rmd

We have focused on the browser-based chatbots for our examples in this course for ease of demonstration.

### Computing Environment Needs

All the examples and activities in this course can be done in the computer environment you prefer. None of the tasks require a huge amount of computing power, so local installations on your personal computer are fine, though everything will also translate to a high performance computing cluster or a cloud computing platform.

In addition to the CodeWhisperer chatbot available on the AWS cloud platform, a variety of cloud-based AI tools and platforms exist that can be leveraged for practical projects after this course. For instance, Amazon SageMaker or Google Cloud AI Platform can be used for constructing and training machine learning models, while Microsoft Azure Cognitive Services can be applied to incorporate natural language processing features into your applications. If you'd like to learn more about how AI can be integrated with cloud computing, you can read about it in the Appendix.

### Boxes

> Throughout this course, you will see several types of specially-colored boxes. In particular, there are two to which you should pay special attention. One box highlights potential ethical issues with an AI usage.

Author: ```suggestion
Throughout this course, you will see several types of specially-colored boxes. In particular, there are two that require special attention. One box highlights potential ethical issues with an AI usage.
```
